python/data/ArtusConfigs/Run2LegacyAnalysis/2016/et.py

```diff
@@ -268,7 +283,7 @@ def build_config(nickname, **kwargs):
   if not isData and not isEmbedded:                 config["Processors"].append( "producer:RooWorkspaceWeightProducer")
   if isEmbedded:                 config["Processors"].append( "producer:EmbeddedWeightProducer")
   if isEmbedded:                 config["Processors"].append( "producer:TauDecayModeWeightProducer")
-  #if not isData:                 config["Processors"].append( "producer:TauTrigger2017EfficiencyProducer")
```

Comment:
Why is TauTrigger2017EfficiencyProducer not applied for 2016 in et, mt & tt? Do we have a different setup there?

Reply:
As far as I know we had a different setup so far. I think we will be switching to a more consistent setup when including the embedding tau trigger efficiencies.

Reply:
ok fine with me